src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php

```diff
@@ -219,6 +219,6 @@
             ])
 
         ->set('serializer.normalizer.backed_enum', BackedEnumNormalizer::class)
-            ->tag('serializer.normalizer', ['priority' => -915])
+            ->tag('serializer.normalizer', ['priority' => -880])
```

Comment:
A comment telling why `880` would deter breaking it in the future. A test case would be even more preventive (something like https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SerializerTest.php)

Reply:
test added